server/manager/src/main/java/org/apache/accumulo/manager/Manager.java

```diff
@@ -1273,6 +1323,10 @@ boolean canSuspendTablets() {
       throw new IllegalStateException("Exception updating manager lock", e);
     }
 
+    // TODO shut this down later? Is this the correct place to start this?
+    this.splitter = new Splitter(context, DataLevel.USER, this);
```

Comment:
Is there a reason not to make this part of the TabletGroupWatcher?

Reply:
Not that I know of, I did not start there because I was unsure if that was the best path and I wanted to get some basic functionality working.  I think that would be a good follow on issue to explore.